src/operator/contrib/modulated_deformable_convolution-inl.h

```diff
@@ -0,0 +1,547 @@
+/*!
```

Comment:
the headers here must follow the same disclaimers in other files: e.g.
https://github.com/apache/incubator-mxnet/blob/master/src/operator/batch_norm_v1-inl.h